scripts/json_add_image_info.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import hashlib
+
+
+def e(variable, default=None):
+    return os.environ.get(variable, default)
+
+
+json_path = "{}{}{}.json".format(e("BIN_DIR"), os.sep, e("IMAGE_PREFIX"))
+
+with open(os.path.join(e("BIN_DIR"), e("IMAGE_NAME")), "rb") as image_file:
+    image_hash = hashlib.sha256(image_file.read()).hexdigest()
+
+
+def get_titles():
+    titles = []
+    for prefix in ["", "ALT0_", "ALT1_", "ALT2_"]:
+        title = {}
+        for var in ["vendor", "model", "variant"]:
+            if e("DEVICE_{}{}".format(prefix, var.upper())):
+                title[var] = e("DEVICE_{}{}".format(prefix, var.upper()))
+
+        if title:
+            titles.append(title)
+
+    if not titles:
+        titles.append({"title": e("DEVICE_TITLE")})
+
+    return titles
```

Comment:
Again, this needs to be adjusted for missing ALTx.

I am aware that it's working anyway (by accident) at the moment, but I'd like it to match the setup in 19.07 more closely.

Reply:
When I wrote the script I tried to have something backward compatible so we don't end up maintaining multiple versions of the same script, however one may also call this an accident :smirk: